<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZrmDevMethods</class>
 <widget class="QWidget" name="ZrmDevMethods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="ZrmMethodsTree" name="spr_methods" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bDownload">
       <property name="toolTip">
        <string>Получить из устройства</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="zrm_resource.qrc">
         <normaloff>:/zrm/icons/arrow_down.png</normaloff>:/zrm/icons/arrow_down.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bAdd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Добавить</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="zrm_resource.qrc">
         <normaloff>:/zrm/icons/right.png</normaloff>:/zrm/icons/right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Убрать</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="zrm_resource.qrc">
         <normaloff>:/zrm/icons/left.png</normaloff>:/zrm/icons/left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bUpload">
       <property name="toolTip">
        <string>Загрузить в устройство</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="zrm_resource.qrc">
         <normaloff>:/zrm/icons/arrow_up.png</normaloff>:/zrm/icons/arrow_up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="dev_methods">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Метод</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZrmMethodsTree</class>
   <extends>QWidget</extends>
   <header>zrmmethodstree.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bDownload</tabstop>
  <tabstop>bAdd</tabstop>
  <tabstop>bRemove</tabstop>
  <tabstop>bUpload</tabstop>
  <tabstop>dev_methods</tabstop>
 </tabstops>
 <resources>
  <include location="zrm_resource.qrc"/>
 </resources>
 <connections/>
</ui>
